src/components/Settings/Subscribe/Subscribe.helpers.js

```diff
@@ -40,10 +50,8 @@ const findVerifiedPurchase = (products, filter) => {
 const findLocalTransaction = (products, filter) => {
   // find if some of those products are part of a receipt
   for (const product of products) {
-    console.log(product);
-    const transaction = window.CdvPurchase.store.findInLocalReceipts(product);
-    //const transaction = findInLocalReceipts(product);
-    console.log('transaction in local receipt', transaction);
+    //const transaction = window.CdvPurchase.store.findInLocalReceipts(product);
+    const transaction = findInLocalReceipts(product);
```

Comment:
The onlies functions of this file that are used `formatDuration` and `formatTitle`. Remove unnecessary stuff